### DISCUSSION TITLE
Winget : publish  package to winget (windows package manager)

### DISCUSSION THREAD

**Discussion by manudss** [lencx/ChatGPT#129] [Ideas]
It would be great to publish a version on Winget, to be able to install it with the new windows command: winget.

And that it is available on the winstall.app site

**Comment by manudss** [comment]
PR created here to publish : https://github.com/microsoft/winget-pkgs/pull/92812 

After, when new version, should rerun command : https://github.com/microsoft/winget-create/blob/main/doc/update.md 

`wingetcreate.exe update --urls <InstallerUrl1>|x64 --version <Version> lencx.ChatGPT`

Maybe if we could implement this, in the release script (if exist should be great !).

**Reply by lencx** [reply]
@manudss Thanks for the suggestion, I will improve it step by step, I am currently working on a larger feature and I will implement this suggestion once it is completed.

**Reply by yixinBC** [reply]
> @manudss Thanks for the suggestion, I will improve it step by step, I am currently working on a larger feature and I will implement this suggestion once it is completed.

That's not a suggestion,that's what @manudss had done for this project!🤣

**Reply by lencx** [reply]
@manudss  Sorry, I misunderstood, thanks for the contribution.

**Reply by manudss** [reply]
Yes at first it was a suggestion, then I looked at how to do it. So I did the PR to add it. It's in progress, it should be validated.

But in fact, we'll have to do it again for each update. And so, yes, we'll have to see how to add it to the release script, to generate the winget version for each new release.

**Reply by yixinBC** [reply]
there are already some github-actions scripts that we can use, so I think we can solve it in the following several days.

**Comment by yixinBC** [comment]
I just open an issue(#142 ) to track this topic and now we can discuss our progress there😘
@manudss @lencx
